docs/photoniq/fingerprinting/index.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Digital Fingerprinting
+---
+
+PhotonIQ Digital Fingerprinting is a managed service that generates persistent digital signatures and visitor IDs for your website users.
+
+## How It Works
+
+Digital Fingerprinting uses advanced fingerprinting techniques to create a unique signature of a user's device. This allows recognizing users across sessions and devices without relying only on cookies.
+
+Some key techniques used:
+
+- Collecting browser signals such as user agent, resolution, fonts, and so on.
+- Leveraging Canvas and AudioContext APIs.
+- Fingerprinting using CPU and Memory.
```

Comment:
I need to ask about "using CPU and Memory".  Of course we are using CPU & memory.  But I think something else was meant here.

Reply:
I thought we were fingerprinting based on customer CPU and Memory numbers

Reply:
CPU & memory are  one of many items that make up the fingerprint.

When I read fingerprinting "using" I thought the sentence was reading that "we generate fingerprints using a cpu and memory" when the fingerprint is calculated using information of the cpu and the memory and many other items.

This might be one I am overthinking again....

Reply:
Updated, how is this?

Reply:
Sounds good.